core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -168,6 +236,50 @@ protected void doStop() throws Exception {
     return server;
   }
 
+  private static String[] getList(String original) {
```

Comment:
@ijuma suggested this earlier, too. I looked at the code for `getList` and it looked wrong to me. However, I just tested it and it works as expected. So I'll move to the `LIST` type.